app/components/award_details/view.rb

```diff
@@ -11,7 +11,11 @@ def initialize(trainee)
     end
 
     def award_date
-      date_for_summary_view(trainee.awarded_at)
+      if trainee.awarded_at.present?
```

Comment:
can this be achieved with `trainee.awarded?`

Reply:
should be achievable that way but I just checked `select count(*) from trainees where state = 6 and awarded_at is null;` on prod and it returns 33 so it's (somehow) possible to have `awarded` state without an `awarded_at` value.